e2e/tests-dfx/install.bash

```diff
@@ -171,3 +171,13 @@ teardown() {
     assert_command dfx canister install e2e_project_backend
     assert_command timeout -s9 20s dfx canister install e2e_project_backend --mode reinstall -y # if -y does not work, hangs without stdin
 }
+
+@test "--no-asset-upgrade skips asset upgrade" {
+    dfx_start
+    use_asset_wasm 0.12.1
+    dfx deploy
+    use_default_asset_wasm
+    assert_command dfx canister install e2e_project_frontend --mode upgrade --no-asset-upgrade
```

Comment:
I would have expected this test to also verify the behavior of `dfx deploy --no-asset-upgrade` @adamspofford-dfinity